src/App.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: #2b58b0;
```

Comment:
I don't like the colour that much, should we consider changing it?

Reply:
Go fuck yourself